### DISCUSSION TITLE
Mass download from a Playlist

### DISCUSSION THREAD

**Discussion by jryltn** [qnblackcat/uYouPlus#1065] [Ideas]
Hi,

As the title suggest, would it be possible to implement mass download of (music) videos from a playlist?
Much like how Spotify users are able to mass download songs to their devices with a click of a button.

Thanks.

**Comment by yodaluca23** [comment]
Ask MiRO on [his repo](https://github.com/MiRO92/uYou-for-YouTube) for uYou.
